admin/app/views/workarea/admin/search_settings/show.html.haml

```diff
@@ -10,45 +10,62 @@
-            %span.property__note= t('workarea.admin.form.csv_field_note')
+              .section.align-center
+                %span= t('workarea.admin.search_settings.show.synonyms.simple_example_description')
+                %pre
+                  :preserve
+                    #{t('workarea.admin.search_settings.show.synonyms.simple_example')}
 
-          = render 'workarea/admin/facets/price_inputs', facet: @price_facets
+              .section.align-center
+                %span= t('workarea.admin.search_settings.show.synonyms.phrase_example_description')
+                %pre
+                  :preserve
+                    #{t('workarea.admin.search_settings.show.synonyms.phrase_example')}
 
         .tabs__panel
           %h2.tabs__heading= t('workarea.admin.search_settings.show.boosts.title')
-          .grid
-            .grid__cell.grid__cell--50
-              %h3= t('workarea.admin.search_settings.show.boosts.product_popularity')
-              %p= t('workarea.admin.search_settings.show.boosts.product_popularity_description')
-              = text_field_tag 'views_factor', @settings.views_factor, class: 'text-box text-box--small'
+          .section
+            .grid
+              .grid__cell.grid__cell--50.align-center
+                %h3= t('workarea.admin.search_settings.show.boosts.product_popularity')
+                %p= t('workarea.admin.search_settings.show.boosts.product_popularity_description')
+                .grid.grid--middle
+                  .grid__cell.grid__cell--10.align-right= t('workarea.admin.search_settings.show.boosts.disabled')
+                  .grid__cell.grid__cell--50= range_field_tag 'views_factor', @settings.views_factor, min: 0, max: 2, step: 0.01, class: 'text-box text-box--wide'
+                  .grid__cell.grid__cell--10.align-left= t('workarea.admin.search_settings.show.boosts.max')
 
-            .grid__cell.grid__cell--50
-              %h3= t('workarea.admin.search_settings.show.boosts.field_weight')
-              %p= t('workarea.admin.search_settings.show.boosts.field_weight_description')
-              - @settings.boosts.each do |field, value|
-                .property
-                  = label_tag "boosts[#{field}]", field.to_s.titleize, class: 'property__name'
-                  = text_field_tag "boosts[#{field}]", value, class: 'text-box text-box--small'
+              .grid__cell.grid__cell--50.align-center
+                %h3= t('workarea.admin.search_settings.show.boosts.field_weight')
+                .grid
+                  .grid__cell.grid__cell--50
+                    %p= t('workarea.admin.search_settings.show.boosts.field_weight_description')
+                - @settings.boosts.each do |field, value|
+                  .property
+                    = label_tag "boosts[#{field}]", field.to_s.titleize, class: 'property__name'
+                    = range_field_tag "boosts[#{field}]", value, min: 0, max: 5, step: 0.1, class: 'text-box text-box--medium'
```

Comment:
This all started with wanting to use these ranges to limit how far people adjust the boosts.

Reply:
Not sure if you meant to do this, but the `<label>` tags are a lot longer than the `<input>` tags they're for:

<img width="471" alt="Screen Shot 2020-10-19 at 3 38 01 PM" src="https://user-images.githubusercontent.com/113026/96503345-15915980-1221-11eb-9cfd-d25f48be6c5f.png">

I noticed this by accidentally clicking some whitespace to the right of the label tag, and saw one of the `<input>`s suddenly became selected. This could probably be avoided by ensuring the width of the label is the same as the width of the input (like by wrapping all these in a single `<div>` with a max-width or something). Not sure if it's a "dealbreaker" because I can't seem to accidentally change anything, so it might just look odd but not actually cause any problems...

Reply:
I don't think that's a big deal, we can revisit later if it causes some grief.